apps/voting/contracts/Voting.sol

```diff
@@ -418,4 +600,32 @@ contract Voting is IForwarder, AragonApp {
         uint256 computedPct = _value.mul(PCT_BASE) / _total;
         return computedPct > _pct;
     }
+
+    function _storeVote(Vote storage vote_, uint256 _voteId, bool _supports, address _voter) private {
```

Comment:
_Super_ nitpick, but perhaps we can name this `_castVote`? Store sounds like it's just manipulating storage, but "cast" sounds like an action that could also (and it does) emit an event.

Reply:
way better! thx :)